internal/exeutils/exeutils.go

```diff
@@ -179,4 +179,8 @@ func DecodeCmd(cmd string) (string, []string) {
 	default:
 		return "sh", []string{"-c", cmd}
 	}
-}
\ No newline at end of file
+}
+
+func WinExeExt(path string) string {
+	return path + Extension
+}
```

Comment:
Please drop this and just use `path + exeutils.Extension`. This isn't saving us much and is just obfuscating logic.